summingbird-storm-test/src/main/scala/com/twitter/summingbird/storm/StormTestRun.scala

```diff
@@ -17,28 +17,27 @@
 package com.twitter.summingbird.storm
 
 import com.twitter.algebird.Semigroup
-import backtype.storm.{ Config => BacktypeStormConfig, LocalCluster, Testing }
+import org.apache.storm.{ LocalCluster, Config => BacktypeStormConfig }
 import com.twitter.summingbird.online.executor.InflightTuples
-import backtype.storm.testing.{ CompleteTopologyParam, MockedSources }
+import org.apache.storm.testing.{ CompleteTopologyParam, MockedSources }
 import com.twitter.summingbird.storm.spout.TraversableSpout
 import com.twitter.summingbird.online.option._
 import com.twitter.summingbird.option._
 import com.twitter.summingbird._
-import com.twitter.summingbird.planner._
-import com.twitter.tormenta.spout.Spout
-import scala.collection.JavaConverters._
 import java.security.Permission
 import com.twitter.util.Duration
+import org.apache.storm.utils.Time
 
 /**
  * This stops Storm's exception handling triggering an exit(1)
  */
 private[storm] class MySecurityManager extends SecurityManager {
   override def checkExit(status: Int): Unit = {
-    throw new SecurityException();
+    throw new SecurityException()
   }
-  override def checkAccess(t: Thread) = {}
-  override def checkPermission(p: Permission) = {}
+  override def checkAccess(t: Thread): Unit = {}
+  override def checkPermission(p: Permission): Unit = {}
+  override def checkPermission(perm: Permission, context: scala.Any): Unit = {}
```

Comment:
please use `AnyRef` not `scala.Any` in this case. `AnyRef` is the correct corresponding type to `java.lang.Object`.